components/automate-cli/cmd/chef-automate/license.go

```diff
@@ -410,6 +419,12 @@ func validateArgs() {
 		fmt.Println("End time cannot be before start time")
 		os.Exit(1)
 	}
+	fileExtension := strings.Split(CommandFlags.FileName, ".")
+
+	if len(fileExtension) == 2 && fileExtension[1] != "csv" {
```

Comment:
Remove this check. The file extension is not a necessary parameter in Linux and we should not add unnecessary restrictions on user.